### PR TITLE
Create repos

### PR DESCRIPTION
**Pull request by Laguigue**

### CODE REVIEW

File: src/components/pages/Create.vue [hunk 1]
```diff
@@ -46,14 +45,37 @@ export default {
...
+
+      gh.getUser().getProfile().then(function (profileResponse) {
+        console.log(profileResponse.data.login); // Success!
+        const username = profileResponse.data.login;
+        gh.getUser(username).createRepo(options).then(function (createResponse) {
+          const remoteRepo = gh.getRepo(username, createResponse.data.name);
+          const obj = {
+            title: 'github-for-fashion',
+          };
+          remoteRepo.writeFile('master', 'garment-config.json', JSON.stringify(obj), 'Garment project setup', function () {}).then(function (repoResponse) {
```
Comment (DavidBruant) [inline src/components/pages/Create.vue]: * Use a variable for `'garment-config.json'`
* what is the empty callback for?
* new line for the `.then`

File: src/components/pages/Create.vue [hunk 2]
```diff
@@ -46,14 +45,37 @@ export default {
...
+            console.log('success');
+            console.log(repoResponse);
+          }, function () {
```
Comment (DavidBruant) [inline src/components/pages/Create.vue]: add an explicit `.catch`

Comment (DavidBruant) [reply src/components/pages/Create.vue]: (several times below)

File: src/components/pages/Create.vue [hunk 3]
```diff
@@ -46,14 +45,37 @@ export default {
...
+      });
+      const options = {
+        name: this.garment.title,
+        description: this.garment.title,
+        homepage: 'http://example.com',
+        auto_init: true,
+        AcceptHeader: 'mercy-preview+json',
+      };
+
+      gh.getUser().getProfile().then(function (profileResponse) {
```
Comment (DavidBruant) [inline src/components/pages/Create.vue]: return to forward promise result (or error)

File: src/components/pages/Create.vue [hunk 4]
```diff
@@ -46,14 +45,37 @@ export default {
...
+        console.log(profileResponse.data.login); // Success!
+        const username = profileResponse.data.login;
+        gh.getUser(username).createRepo(options).then(function (createResponse) {
```
Comment (DavidBruant) [inline src/components/pages/Create.vue]: return to forward promise result (or error)

File: src/components/pages/Create.vue (hunk 1 above)
Comment (DavidBruant) [reply src/components/pages/Create.vue]: return to forward promise result (or error)

File: src/components/pages/Create.vue [hunk 5]
```diff
@@ -46,14 +45,37 @@ export default {
...
-      EventBus.$emit('showError', error);
+      const gh = new GitHub({
+        token: 'insert your token here',
+      });
+      const options = {
+        name: this.garment.title,
+        description: this.garment.title,
+        homepage: 'http://example.com',
+        auto_init: true,
+        AcceptHeader: 'mercy-preview+json',
```
Comment (DavidBruant) [inline src/components/pages/Create.vue]: still necessary? If so add a comment to explain why

**Review by DavidBruant — CHANGES_REQUESTED** [review]

File: src/components/pages/Create.vue (hunk 5 above)
Comment (Laguigue) [reply src/components/pages/Create.vue]: Useless indeed

File: src/components/pages/Create.vue (hunk 3 above)
Comment (Laguigue) [reply src/components/pages/Create.vue]: c'est good

File: src/components/pages/Create.vue (hunk 4 above)
Comment (Laguigue) [reply src/components/pages/Create.vue]: c'est good

**Pull request merged**